tests/test_utils.py

```diff
@@ -219,7 +219,10 @@ def test_is_absolute_href(self) -> None:
             ("item.json", False),
             ("./item.json", False),
             ("../item.json", False),
-            ("/item.json", True),
+            # https://docs.python.org/3/library/os.path.html#os.path.isabs
+            # Windows requires a drive name.
+            ("/item.json", os.name != "nt"),
+            ("d:/item.json", os.name == "nt"),
```

Comment:
Let me know if it's worth splitting the tests out into windows and non-windows (I saw `    @pytest.mark.skipif(os.name != "nt", reason="Windows only test"`  just below but this seemed fine)